src/Calculator/Views/GraphingCalculator/GraphingCalculator.xaml

```diff
@@ -411,246 +448,257 @@
+                                    <SolidColorBrush x:Key="RepeatButtonBackgroundPressed" Color="#20000000"/>
+                                    <CornerRadius x:Key="TopButtonCornerRadius">4,4,0,0</CornerRadius>
+                                    <CornerRadius x:Key="BottomButtonCornerRadius">0,0,4,4</CornerRadius>
+                                    <CornerRadius x:Key="LeftButtonCornerRadius">4,0,0,4</CornerRadius>
+                                    <CornerRadius x:Key="RightButtonCornerRadius">0,4,4,0</CornerRadius>
+                                    <Style x:Key="ThemedGrapherStyle" TargetType="graphControl:Grapher">
+                                        <Setter Property="AxesColor" Value="Black"/>
+                                        <Setter Property="GraphBackground" Value="White"/>
+                                    </Style>
+                                </ResourceDictionary>
+                                <ResourceDictionary x:Key="Dark">
+                                    <CornerRadius x:Key="TopButtonCornerRadius">4,4,0,0</CornerRadius>
+                                    <CornerRadius x:Key="BottomButtonCornerRadius">0,0,4,4</CornerRadius>
+                                    <CornerRadius x:Key="LeftButtonCornerRadius">4,0,0,4</CornerRadius>
+                                    <CornerRadius x:Key="RightButtonCornerRadius">0,4,4,0</CornerRadius>
+                                    <Style x:Key="ThemedGrapherStyle" TargetType="graphControl:Grapher">
+                                        <Setter Property="AxesColor" Value="White"/>
+                                        <Setter Property="GraphBackground" Value="Black"/>
+                                    </Style>
+                                </ResourceDictionary>
+                                <ResourceDictionary x:Key="HighContrast">
+                                    <CornerRadius x:Key="TopButtonCornerRadius">0</CornerRadius>
+                                    <CornerRadius x:Key="BottomButtonCornerRadius">0</CornerRadius>
+                                    <CornerRadius x:Key="LeftButtonCornerRadius">0</CornerRadius>
+                                    <CornerRadius x:Key="RightButtonCornerRadius">0</CornerRadius>
+                                    <Style x:Key="ThemedGrapherStyle" TargetType="graphControl:Grapher">
+                                        <Setter Property="AxesColor" Value="{ThemeResource SystemColorWindowTextColor}"/>
+                                        <Setter Property="GraphBackground" Value="{ThemeResource SystemColorWindowColor}"/>
+                                    </Style>
+                                </ResourceDictionary>
+                            </ResourceDictionary.ThemeDictionaries>
+                        </ResourceDictionary>
+                    </Grid.Resources>
+                    <graphControl:Grapher Name="GraphingControl"
```

Comment:
Not sure if related to this PR, but when on a single pane the graph surface is zoomed in and offset on the default view
![image](https://user-images.githubusercontent.com/6334170/76481559-0adf2e00-63cf-11ea-91f3-6e41e8c02113.png)

Reply:
Yes, this is an issue with the GraphControl we will need to investigate (other PR)